lib/matplotlib/backends/backend_pgf.py

```diff
@@ -77,23 +58,20 @@ def _parse_lualatex_version(output):
 def get_fontspec():
     """Build fontspec preamble from rc."""
     latex_fontspec = []
-    texcommand = get_texcommand()
+    texcommand = rcParams["pgf.texsystem"]
```

Comment:
Do we explicitly not want the correction and fallback mechanism to xelatex, which was implemented in `get_texcommand`?

Reply:
That does already get checked during standard rcparam validation.